backend/src/main/java/net/finman/mapper/CustomerMapper.java

```diff
@@ -0,0 +1,19 @@
+package net.finman.mapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.springframework.jdbc.core.RowMapper;
+import net.finman.model.UserCustomer;
+
+public class CustomerMapper implements RowMapper<UserCustomer> {
+
+    @Override
+    public UserCustomer mapRow(ResultSet rs, int rowNum) throws SQLException {
+        UserCustomer userCustomer = new UserCustomer();
+        userCustomer.setUser(rs.getString("email"));
+        userCustomer.convertAndSetCustomerJsonToObject(rs.getString("customer"));
```

Comment:
Yeah, I don't why we do this, but we keep converting JSON to java objects and back to JSON again